packages/esm-appointments-app/src/form/appointments-form.component.tsx

```diff
@@ -276,10 +276,10 @@ const AppointmentsForm: React.FC<AppointmentsFormProps> = ({
 
     // check if Duplicate Response Occurs
     const response: FetchResponse = await checkAppointmentConflict(appointmentPayload);
-    let errorMessage = t('appointmentDuplicateForm', 'Appointment is not available');
-    if (response?.data?.SERVICE_UNAVAILABE) {
+    let errorMessage = t('appointmentConflict', 'Appointment conflict');
+    if (response?.data?.hasOwnProperty('SERVICE_UNAVAILABLE')) {
```

Comment:
I had to change it to this format to get things to work for me for some reason... was this working for you previously @senthil-k8s ?

Reply:
@mogoodrich yeah, it works.

Reply:
Strange @senthil-k8s ... not sure why it wasn't working for me but working for you previously... but are you okay with this change (assuming I get the E2E test fixed)...